daemon/snap.go

```diff
@@ -55,27 +55,27 @@ func snapDate(info *snap.Info) time.Time {
 }
 
 // localSnapInfo returns the information about the current snap for the given name plus the SnapState with the active flag and other snap revisions.
-func localSnapInfo(st *state.State, name string) (info *snap.Info, active bool, err error) {
+func localSnapInfo(st *state.State, name string) (info *snap.Info, snapstOut *snapstate.SnapState, err error) {
```

Comment:
Can we please name it `snapst` instead of `snapstOut`?

Btw, curious that the documentation is right for the change. Mind-reading? :)